documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -1178,6 +1258,18 @@ The following _advanced_ configuration properties have defaults that work in mos
 The operations include: `c` for inserts/create, `u` for updates, and `d` for deletes.
 By default, no operations are skipped.
 
+|[[vitess-property-provide-transaction-metadata]]<<vitess-property-provide-transaction-metadata, `provide.transaction.metadata`>>
+|`false`
+|Determines whether the connector generates events with transaction boundaries and enriches change event envelopes with transaction metadata. Specify `true` if you want the connector to do this. See {link-prefix}:{link-vitess-connector}#vitess-transaction-metadata[Transaction metadata] for details.
```

Comment:
Could you use the xref syntax (if there's no example for that in this doc, you can find it in others, e.g. the PG one) instead of link-prefix? The latter is a left-over from a time where we couldn't use xref for some issue when rendering downstream product docs; but this has been resolved and link-prefix should be phased out in favour of the more concise xref. I know it's not handled consistently like that yet, but let's avoid adding more link-prefix usages in new/changed docs.

Reply:
Thanks for calling this out. I'm not really familiar of the xref syntax and not sure what part should be changed.
I checked https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/pages/connectors/postgresql.adoc#L3130  and it seems it's also using link-prefix for the counterpart config section on transaction metadata: `{link-prefix}:{link-postgresql-connector}#postgresql-transaction-metadata[Transaction metadata]`.

It would be great if there's a how-to guide or canonical example to follow. We might as well want to have a refactoring pass to fix docs consistently

Reply:
You can find an example here: https://github.com/debezium/debezium/blame/main/documentation/modules/ROOT/pages/connectors/postgresql.adoc#L343. I.e. it's `xref:reference-id[]`, or `xref:reference-id[some text]`. More details in the [Antora docs](https://docs.antora.org/antora/3.0/page/xref/).

Reply:
> We might as well want to have a refactoring pass to fix docs consistently

Yeah, if you wanted to change this completely for the Vitess docs, that'd be great.